pkg/cmd/cli/cli.go

```diff
@@ -208,9 +202,20 @@ func NewCommandCLI(name, fullName string, in io.Reader, out, errout io.Writer) *
 	}
 	cmds.AddCommand(cmd.NewCmdOptions(out))
 
+	setOutput(cmds, out)
+
 	return cmds
 }
 
+func setOutput(cmd *cobra.Command, out io.Writer) {
```

Comment:
Cascading the writer to the entire command tree would allow us, for example, to extend the use of the responsive writer for *every* command output (not just help) by setting it only in the root cmd. 

Although it makes sense at a first glance, there are places (e.g. tables in `get`) where using the max width possible also probably makes sense, so I ended up not using this and only enabling the responsive writer in help.

I'll remove this block and let us set it individually in commands where we want it.